docs/src/pages/conversations.mdx

```diff
@@ -725,3 +725,64 @@ The `agent_error` event is sent whenever an error occured durint the AgentMessag
 
   </Col>
 </Row>
+
+---
+
+## Add a new Content Fragment {{ tag: 'POST', label: '/v1/w/:workspace_id/assistant/conversations/:conversation_id/content_fragments' }}
+
+<Row>
+  <Col>
+
+    This endpoint enables you to create a new content fragment in a conversation.
```

Comment:
A bit of explanation + diff to the intro text of the page to mention that it can be a USerMessage a AgentMessage or a ContentFragment?

Reply:
Added